src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -197,12 +205,24 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
     with self.command_group('acr repository', acr_repo_util) as g:
         g.command('list', 'acr_repository_list')
         g.command('show-tags', 'acr_repository_show_tags')
-        g.command('show-manifests', 'acr_repository_show_manifests')
+        g.command('show-manifests', 'acr_repository_show_manifests',
+                  deprecate_info=self.deprecate(redirect='acr manifest metadata list', hide=True))
         g.show_command('show', 'acr_repository_show')
         g.command('update', 'acr_repository_update')
         g.command('delete', 'acr_repository_delete')
         g.command('untag', 'acr_repository_untag')
 
+    with self.command_group('acr manifest', acr_manifest_util, is_preview=True) as g:
+        g.show_command('show', 'acr_manifest_show')
+        g.command('list', 'acr_manifest_list', table_transformer=manifest_output_format)
+        g.command('delete', 'acr_manifest_delete')
```

Comment:
Do we need to consider adding `confirmation=True` for the delete command?

Reply:
No, we have a custom confirmation function/message in the source (line 446 of manifest.py). This is the same thing we use for `az acr repository delete` as well which is why that command also doesn't have this parameter (line 212 of commands.py)

Reply:
OK, got it